content/3_Silver/Graph_Traversal.mdx

```diff
@@ -1210,46 +1253,38 @@ public class BuildingTeams {
 <PySection>
 
 ```py
+import sys
 from collections import deque
 
-n, m = map(int, input().split())
+input = sys.stdin.readline
```

Comment:
same thing here

Reply:
It barely passes in the BFS version. Should I include or exclude fast input?
Fast Input: https://cses.fi/problemset/result/11118910/
No Fast Input: https://cses.fi/problemset/result/11126773/